sensor/docs/how_to_intrinsic_camera_calibrator.md

```diff
@@ -0,0 +1,229 @@
+- A good detection should be able to be fitted correctly with a camera intrinsic calibrator. For this reason, we also calibrate the camera intrinsics with each detection, to evaluate how good the detection is by observing the reprojection error produced by this calibration.
+
+#### 2.4.2 Redundant detection rejection
+
+In addition to filtering detections based only on their statistics, it is also necessary to consider the statistics of the database as a whole in order to avoid redundant data, which may induce unbalanced distributions.
+
+In other to add a detection to the database, we impose an innovation threshold with respect to the elements on the database in at least one of the following aspects:
+
+- board center (in pixels)
+- board size (in pixels)
+- board distance from the camera
+- board orientation angle from the camera's z-plane
+
+#### 2.4.2 Data collection visualization
+
+In order to understand the data collection process, and guide the user to place the calibration board in positions where samples are needed, we implement a heatmap visualization that shows the number of board points spatially. which can be enabled by clicking the `Draw training occupancy` and selecting an appropriate `Drawings alpha` value.
+
+Figure 12 shows an example where few samples have been collected and Figure 13 shows an example where more samples have been collected. In order to improve the database, the user should move the board to areas in which the heatmap is displayed in blue.
+
+| ![data_collection_heatmap_1.png](images/camera-intrinsics/data_collection_heatmap_1.jpg) | ![data_collection_heatmap_2](images/camera-intrinsics/data_collection_heatmap_2.jpg) |
+| :--------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------: |
+|                      Fig 12. Data collection heatmap (few samples)                       |                           Fig 13. Data collection heatmap                            |
+
+The status of the data collection can also be observed in the main window, where the number of samples collected and the occupancy rate of the previously mentioned heatmaps is displayed (See Figure 14).
+
+<p align="center">
+   <img src="images/camera-intrinsics/data_collection_results.png" width="200" class="center">
+   <p align="center">Fig 14. Detection results</figcaption>
+</p>
+
+Finally, we also provide a visualization for the database statistics, which can be enabled by clicking the `View data collection statistics` (an example can be observed in Figure 15).
+
+<p align="center">
+   <img src="images/camera-intrinsics/data_collection_statistics.png" width="80%" class="center">
+   <p align="center">Fig 15. Detection results</figcaption>
+</p>
+
+### 2.5 Calibration and visualization
```

Comment:
Could you please add the instruction about file output and its format? It's better with some reference links, like http://wiki.ros.org/image_pipeline/CameraInfo.

Reply:
Added in [a2a138c](https://github.com/tier4/CalibrationTools/pull/65/commits/a2a138c88c8bc9d3ae3ee8d6cd68256adc1637b4)